app/assets/stylesheets/dashboard/bookings/_dashboard_booking.scss

```diff
@@ -1,38 +1,45 @@
 .dashboard{
-  background-color: brown;
+  // background-color: brown;
+  background-image: image-url("greyshells.jpg");
   background-attachment: scroll;
   background-size: 100%;
+  height: 650px;
 }
 
 .table {
-  background: orange;
+  background: #64868E;
   border: 2px, solid, black;
-  margin: 20px;
+  align-items: center;
+  color: black;
 }
 
 .split {
- background: yellow;
+ background: #F3FBF1;
+ color: black;
 }
 
 th{
```

Comment:
Pay attention to not work on default tags, for example "th", "td" but put a class ;)